docs/legacy/configuring.asciidoc

```diff
@@ -21,7 +21,6 @@ include::{libbeat-dir}/shared/configuring-intro.asciidoc[]
 * <<configuration-path>>
 * <<configuration-rum>>
 * <<configuration-ssl-landing>>
-* <<transaction-metrics>>
```

Comment:
**issue**: this needs to be backported to 7.x as well

Reply:
Yup. I updated the backport labels. We don't usually backport doc changes to previous versions of 8.x, but I can if y'all think that's necessary in this case.